src/View/Parsers/Diff.php

```diff
@@ -161,15 +161,21 @@ public static function compareHTML($from, $to, $escape = false)
     }
 
     /**
-     * @param string|array $content If passed as an array, values will be concatenated with a comma.
+     * @param string|bool|array $content If passed as an array, values will be concatenated with a comma.
      * @return array
      */
     public static function getHTMLChunks($content)
     {
+        if (is_bool($content)) {
+            // Convert boolean to strings
+            $content = $content ? "true" : "false";
+        }
+
         if ($content && !is_string($content) && !is_array($content) && !is_numeric($content)) {
             throw new InvalidArgumentException('$content parameter needs to be a string or array');
```

Comment:
Could we throw the `is_bool()` check to this `InvalidArgumentException`? Could move the conversion to below this, and I think this would be a good addition.